timezones.json

```diff
@@ -53,7 +53,7 @@
   {
     "value": "Pacific Standard Time (Mexico)",
     "abbr": "PDT",
-    "offset": -7,
+    "offset": -8,
```

Comment:
https://www.timeanddate.com/time/zones/pdt

Reply:
And so on. Why are you proposing this changes?

Reply:
@dmfilipenko `offset` property values doesn't match the number from `text` property. For example:
http://prntscr.com/i792ph

Reply:
because `isdst` time

Reply:
Closing due unneeded.